storage/tianmu/handler/ha_tianmu.cpp

```diff
@@ -112,7 +112,7 @@ static core::Value GetValueFromField(Field *f) {
       // convert to UTC
       if (!common::IsTimeStampZero(my_time)) {
         my_bool myb;
-        my_time_t secs_utc = current_txn_->Thd()->variables.time_zone->TIME_to_gmt_sec(&my_time, &myb);
+        my_time_t secs_utc = f->table->in_use->variables.time_zone->TIME_to_gmt_sec(&my_time, &myb);
```

Comment:
please add some comments here, I don't quite understand the code changes here.

Reply:
f->table->in_use is to get the THD pionter, current_txn_->Thd()  is also to get the THD pointer. I want to store default value from the function  GetValueFromField, but in some senario, the cureent_txn is not initialized and pointed to null. I find f->table->in_use is not nullptr, so i change the code to  f->table->in_use.